tools/itch5inside.cpp

```diff
@@ -38,13 +39,17 @@ class config : public jb::config_object {
   jb::config_attribute<config, jb::offline_feed_statistics::config>
       symbol_stats;
   jb::config_attribute<config, bool> enable_symbol_stats;
+  jb::config_attribute<config, bool> enable_array_based;
+  jb::config_attribute<config, typename jb::itch5::order_book<
+                                   jb::itch5::array_based_order_book>::config>
+      book_cfg;
 };
 
 } // anonymous namespace
 
-int main(int argc, char* argv[]) try {
-  config cfg;
-  cfg.load_overrides(argc, argv, std::string("itch5inside.yaml"), "JB_ROOT");
+template <typename book_type_t, typename cfg_book_t>
```

Comment:
Please add a doxygen comment describing this function, particularly its template parameters.